src/main/java/uk/ac/cam/cl/dtg/segue/api/managers/ExternalAccountManager.java

```diff
@@ -87,7 +86,7 @@ public synchronized void synchroniseChangedUsers() throws ExternalAccountSynchro
                             //    Action: same as deletion? Or just remove from lists for now?
                             log.debug("Case: delivery failed.");
                             mailjetApi.updateUserSubscriptions(mailjetId, MailJetSubscriptionAction.REMOVE, MailJetSubscriptionAction.REMOVE);
-                        } else if (!accountEmail.equals(mailjetDetails.getString("Email"))) {
+                        } else if (!accountEmail.toLowerCase().equals(mailjetDetails.getString("Email"))) {
```

Comment:
From this, I'm assuming MailJet always stores emails lower case? I guess in the past if a user had upper-case letters in their email each synchronize changed user would have meant the creation of a new user and deletion of another rather than updating the user. As those operations are the same (except for maybe some history loss) we don't have to worry about prior users who incorrectly fell down this branch.